tasks/handle-action-destroy.yaml

```diff
@@ -1,5 +1,17 @@
 ---
 - name: Call Ansible Tower job-runner to destroy {{ anarchy_subject_name }}
+  vars:
+    _params:
```

Comment:
Can we name this var something more descriptive? `_job_vars` or `action_job_vars`?

Reply:
done